Packs/TOPdesk/Integrations/TOPdesk/README.md

```diff
@@ -51,6 +52,11 @@ Make sure you use the right account for your needs and that the account used has
 #### Troubleshooting
 Make sure the application password is not expired by logging in TOPdesk and viewing it as described in step 3 of the **Setup TOPdesk's application password** procedure. 
 
+### Finding the Default Incident Caller ID
+The TOPdesk incident caller is the TOPdesk person who requested a specific TOPdesk incident and is the contact person for all activities related to this incident.  
+To find the incident caller ID which should be used as default caller when creating a new TOPdesk incident, you can first configurate the integration instance *without* providing the **Default Incident Caller ID**. Then run the command `!topdesk-persons-list` in the *Playground - War Room*.  
+You will be returned a list of persons and their IDs. Copy the ID of the desired person and edit the integration instance to set the **Default Incident Caller ID**.
```

Comment:
```suggestion
You will receive a list of people and their IDs. Copy the ID of the desired person and edit the integration instance to set the **Default Incident Caller ID**.
```

Reply:
The term of the TOPdesk entity we are requesting with this API command is actually "persons". I know this is not proper english, but I thought I call it like TOPdesk does in their software.
See the TOPdesk API documentation: https://developers.topdesk.com/explorer/?page=supporting-files#/Persons/retrievePersons